examples/README.md

```diff
@@ -10,10 +10,11 @@ Overview
 | <a href="Authorize requests using external authorizer.policy.xml">Authorize requests using external authorizer</a>                                                                 | Secure API access by using an external authorizer encapsulating custom authentication/authorization logic                                                                                                                             |
 | <a href="Back-end API redundancy.policy.xml">Back-end API redundancy</a>                                                                                                           | Routes calls to the closest of two backend services, and fails over to the secondary if an HTTP 404 is returned.                                                                                                                      |
 | <a href="Call out to an HTTP endpoint and cache the response.policy.xml">Call out to an HTTP endpoint and cache the response</a>                                                   | Enrich requests by calling HTTP endpoint before calling a backend service                                                                                                                                                             |
+| <a href="Create HMAC SHA256-Signed JWT.xml">Create HMAC SHA256-Signed JWT</a>                                                                                                      | Creates a JSON Web Token with an HMAC SHA256 signature based on a secret from APIM Named Values                                                                                                                                       |
```

Comment:
All the other examples are linked to .policy.xml files - can you rename to make consistent?

Reply:
Good call! Fixed.